linux/secureboot_enable_disable/secureboot_enable_disable.yml

```diff
@@ -31,6 +31,20 @@
             skip_reason: "Not Supported"
           when: vm_hardware_version_num | int < 13
 
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: >-
+              Skip test case {{ ansible_play_name }} because secure boot is not supported
+              for {{ ansible_distribution }} {{ ansible_distribution_version }} guest OS
```

Comment:
`{{ ansible_distribution }} {{ ansible_distribution_version }}` can be replaced by `{{ vm_guest_os_distribution }}`.